index.html

```diff
@@ -16,16 +19,139 @@
 
 <body>
 
-    <div class="container">
-      <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+  <div class="container">
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div class="jumbotron">
+          <div class="row">
+            <div class="col-sm-12">
+
+              <nav class="nav">
+
+                <a class="nav-link active" href="#">
+                  <i class="fas fa-user"></i> About Me</a>
+                <a class="nav-link" href="#">
+                  <i class="far fa-keyboard"></i> Experience</a>
+                <a class="nav-link" href="#">
+                  <i class="far fa-folder-open"></i> Portfolio</a>
+                <a class="nav-link" href="#">
+                  <i class="fas fa-phone-square"></i> Contact</a>
+              </nav>
+            </div>
+            <!-- col-sm-12 -->
+          </div>
+          <!-- row -->
+          <h1 class="display-4">Justin M. Rigler
+            <span class="badge badge-secondary">Updated</span>
+          </h1>
+          <img src="img/profile-img.jpg" class="img-fluid" alt="me not at denny's">
```

Comment:
Haha, nice alt message! Your image is not entirely aligned in the container, also can be an easy fix through CSS.